box-ui/src/components/css/BoxHome.css

```diff
@@ -8,6 +8,15 @@
     height: 100vh;
     width: 100vw;
 }
+
+.CreateMeetingComponent {
```

Comment:
This should be in the `BoxMeeting.css` stylesheet and as a matter of fact already is as the class `.BoxMeeting`